src/components/schedule/day.tsx

```diff
@@ -0,0 +1,49 @@
+import { HorizontalRule } from "@/components/horizontal-rule";
+import type { EventProps } from "@/lib/types";
+
+interface Props {
+  event: EventProps;
+  isOn: boolean;
+}
+
+export function Day({ event, isOn }: Props) {
```

Comment:
jestem fanem raczej inline'owania propsów jeśli ich nie eksportujesz

```suggestion
 export function Day({ event, isOn }: {
  event: EventProps;
  isOn: boolean;
}) {
```